shell/components/YamlEditor.vue

```diff
@@ -103,7 +103,7 @@ export default {
         mode:            'yaml',
         lint:            !readOnly,
         lineNumbers:     !readOnly,
-        styleActiveLine: true,
```

Comment:
Although we had this set to true previously, no styling was actually applied to active lines. So to keep parity with 2.9 styling, I disabled this prop

Reply:
Crossed that as well, I'd set it as `false` but that's the default one looks like.